unity-renderer/Assets/Scripts/MainScripts/DCL/Models/AvatarModel/AvatarModel.cs

```diff
@@ -145,10 +150,30 @@ public void CopyFrom(AvatarModel other)
         expressionTriggerTimestamp = other.expressionTriggerTimestamp;
         wearables = new List<string>(other.wearables);
         emotes = other.emotes.Select(x => new AvatarEmoteEntry() { slot = x.slot, urn = x.urn }).ToList();
+        forceRender = new HashSet<string>(other.forceRender);
     }
 
     public override BaseModel GetDataFromJSON(string json) =>
         Utils.SafeFromJson<AvatarModel>(json);
 
-
+    public AvatarModelDTO ToAvatarModelDto()
```

Comment:
The dependency should be inverted. The DTO should be in an outter layer from the model class. The model class should not depend on DTOs. So instead you should do:
```
AvatarModelDTO AvatarModelDTO.FromModel(avatarModel);
AvatarModel model = avatarModelDto.ToAvatarModel();
```

You can postpone this change tho, but please add the TODO comment